scripts/lintcommit.lua

```diff
@@ -45,8 +45,13 @@ end
 -- Returns nil if the given commit message is valid, or returns a string
 -- message explaining why it is invalid.
 local function validate_commit(commit_message)
-  local commit_split = vim.split(commit_message, ":")
+  -- Return nil if the commit message starts with "fixup" as it signifies it's
+  -- a work in progress and shouldn't be linted yet.
+  if vim.startswith(commit_message, "fixup") then
+    return nil
```

Comment:
can we trim `fixup\!?` from the start of the string and lint the rest?

Reply:
That would fail linting when people do fixup to the fixup.

`fixup! fixup! feat: make it go fast`

Reply:
any number of fixups could be trimmed. but we can revisit this later.